.travis.yml

```diff
@@ -19,8 +19,8 @@ matrix:
     env: SUITE=ubuntu
   - rvm: 2.3
     env: SUITE=windows-server-2012r2
-  - rvm: 2.3
-    env: SUITE=windows-nano
+  # - rvm: 2.3
+  #   env: SUITE=windows-nano
```

Comment:
We haven't officially released anything `*.appx` to the `stable` channel. The decision on when to do this should come from @thommay and the community team as they need to be prepared to support it.